scripts/useMaterialUIv6.mjs

```diff
@@ -2,7 +2,14 @@ import childProcess from 'child_process';
 
 const pnpmUpdate = childProcess.spawnSync(
   'pnpm',
-  ['update', '-r', '@mui/material@6.x', '@mui/system@6.x', '@mui/icons-material@6.x'],
+  [
+    'update',
+    '-r',
+    '@mui/material@6.x',
+    '@mui/system@6.x',
+    '@mui/icons-material@6.x',
+    '@mui/utils@6.x',
```

Comment:
I have added the `@mui/utils` bump to make it more correct. 🤔 
I feel like after https://github.com/mui/mui-x/pull/14867 it makes sense. 🤷